cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/VirtualMachineConfigurationDTO.java

```diff
@@ -15,15 +15,18 @@
 
 public class VirtualMachineConfigurationDTO extends NamedEntityDTO
 {
-    @ApiModelProperty( "CPU cores" )
+    @Valid
+    private Set<UrlEntityDTO> instanceTypes;
+
+    @ApiModelProperty( value = "CPU cores", hidden = true )
     @Min( 1 )
     @Max( 128 )
-    @NotNull
+    @Deprecated
     private BigDecimal cpu;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)